README.md

```diff
@@ -12,6 +13,10 @@ The combined effect of light and dark daisies keeps Daisyworld in the habitable
 
 [1] A. J. Watson and J. E. Lovelock, “Biological homeostasis of the global environment: the parable of Daisyworld,” Tellus B, vol. 35B, no. 4, pp. 284–289, Sep. 1983, doi: 10.1111/j.1600-0889.1983.tb00031.x.
 
+### Testing 
+
+TOTAL                                  878    243    72%
+ [coverage.txt](coverage.txt) [coverage.txt](coverage.txt) [coverage.txt](coverage.txt)
```

Comment:
Whoa! The proliferation of links to coverage.txt is not intended behavior!

remove '\n' characters before adding the summary to README.md